style.css

```diff
@@ -0,0 +1,516 @@
+    line-height: 21px;
+  }
+
+  .up-section__btn {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 490px;
+    padding-left: 5px;
+    margin-top: 23px;
+    text-decoration: none;
+    color: #000;
+  }
+
+  .up-section__btn .up-section__btn_offer {
+    position: relative;
+    width: 232px;
+    padding-top: 14px;
+  }
+
+  .up-section__btn_offer:hover {
+    transform: scale(1.1);
+  }
+
+  .up-section__arrow {
+    cursor: pointer;
+    position: relative;
+    bottom: 15px;
+  }
+  .up-section__arrow div {
+    position: relative;
+    top: 20px;
+    width: 60px;
+    height: 2px;
+    background-color: #000;
+    left:0;
+    display: block;
+  }
+
+  .up-section__arrow div::after {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: -3px;
+    right: 0;
+    background-color: #000;
+    transform: rotate(45deg);
+  }
+  .up-section__arrow div::before {
+    content: '';
+    position: absolute;
+    width: 7px;
+    height: 1px;
+    top: 4px;
+    right: 0px;
+    background-color: #000;
+    box-shadow: 0 3px 5px rgba(0, 0, 0, .2);
+    transform: rotate(-45deg);
+  }
+
+  .up-section__scrooll {
+    bottom: 39px;
+    width: 27px;
+    height: 46px;
+  }
+
+  .up-section__scrooll:hover {
+    transform: scale(1.05);
+  }
+
+  .up-section__contacts {
+    margin-top: 69px;
+    margin-bottom: 21px;
+    margin-left: 12px;
+  }
+
+  .social {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    width: 153px;
+    list-style: none;
+  }
+
+  .social__item:hover {
+    transform: scale(1.05);
+  }
+}
+
+@media (min-width: 375px) and (max-width: 992px) {
+  .header__head {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    max-width: 461px;
+    background: #fff;
+  }
+
+  .header__logo {
+    margin-top: 14px;
+    margin-left: 26px;
+    color: #000;
+  }
+
+  .up-section__title {
+    display: flex;
+    justify-content: center;
+    top:5px;
+  }
+
+  .up-section__title::after {
+    position: absolute;
+    display: block;
+    display: flex;
+    justify-content: center;
+    top: 96px;
+    content: '';
+    width: 14px;
+    height: 14px;
+    border: 3px solid #000;
+    border-right-color: transparent;
+    border-top-color: transparent;
+    -webkit-transform: rotate(-44deg);
+    -ms-transform: rotate(-44deg);
+    transform: rotate(-44deg);
+  }
+
+  .up-section__scrooll {
+    bottom: -60px;
+    width: 27px;
+    height: 46px;
+  }
+
+  .header__lang,
+  .up-section__btn,
+  .up-section__scrooll {
+    visibility: hidden;
+  }
+
+  .up-section__collection-image img {
+  width: 100%;
+  height: 100%;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5676737)